site/gatsby-site/src/components/blog/PostPreviewNew.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
The `New` suffix reminds of the `new new final don't delete` naming convention 😆 

Maybe `PostPreviewPrismic` or something like that? Or is it temporal while the entire blog is migrated?